docs/tm/usage.md

```diff
@@ -5,7 +5,7 @@ Deploy service from Docker image
 tm deploy service foo -f gcr.io/google-samples/hello-app:1.0 --wait
 ```
 
-If you have Dockerfile for your service, you can use kaniko buildtemplate to deploy it
+If you have Dockerfile for your service, you can use Kaniko build template to deploy it
```

Comment:
`buildtemplate` is one word in this context due to the knativism.

Reply:
Thanks! I was contemplating this for a while and thought to just hcnage it haha. I will make these changes.